.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,19 +1,21 @@
-<!--  Thanks for sending a pull request!  Here are some tips for you:
-1. Follow the instructions for writing a release note from k8s: https://git.k8s.io/community/contributors/guide/release-notes.md
+<!-- Thanks for sending a pull request!
+Follow the instructions for writing a release note from k8s: https://git.k8s.io/community/contributors/guide/release-notes.md.
 -->
 
-**What this PR does / why we need it**:
+**What are these changes and why are they needed**:
+Describe your changes here.
 
-**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
-Fixes #
+**What issue are you fixing**:
+Fixes #(issue) (Will be closed after merging your changes.)
 
-**Special notes for your reviewer**:
+**Special notes**:
```

Comment:
Would it make sense to align it with kubernetes? https://github.com/kubernetes/kubernetes/blob/master/.github/PULL_REQUEST_TEMPLATE.md

Reply:
Done.